test/error_test.c

```diff
@@ -129,13 +129,8 @@ test_error(hid_t file)
         TEST_ERROR;
     if (old_data != NULL)
         TEST_ERROR;
-#ifdef H5_USE_16_API
```

Comment:
Why is this out?

Reply:
I tried to explain in the commit message. It's dead code. See line 23.

Reply:
Ah, yes, definitely. :-)

Reply:
> I tried to explain in the commit message. It's dead code. See line 23.

BTW - is there a way I view the commit messages from the PR?

Reply:
Yes, click "Commits" at the top of this page, then click the commit itself, should go to a link like https://github.com/HDFGroup/hdf5/pull/448/commits/0d8c542e9133004b8b868609470e2b571b208e04

Reply:
I'm less certain of the `DOXYGEN_SHOULD_SKIP_THIS` change there BTW, was a bit hard to untangle, but the warning was *not* a false positive.